blueflood-core/src/test/java/com/rackspacecloud/blueflood/types/BaseRollupTest.java

```diff
@@ -0,0 +1,17 @@
+package com.rackspacecloud.blueflood.types;
+
+/**
+ * Holds methods shared by many RollupTest classes
+ */
+public class BaseRollupTest {
+
+    protected <T> Points<T> asPoints(Class<T> type, long initialTime, long timeDelta, T... values) {
+        Points<T> points = new Points<T>();
```

Comment:
This methods looks similar to the one in BaseRollupSerializerTest. Are they different?